### DISCUSSION TITLE
backslashes in share name break rendering of settings' system page

### DISCUSSION THREAD

**Discussion by vale-max** [rern/rAudio#1016] [Fixed Issues] [answered]
I was working on windows so I erroneously used \ instead of / in the share name "media\Music\Flac"

This annoyed enough the rendering of the page that reported an erroneously escaped character in JSON data. I had to ssh in and change /etc/fstab manually to fix.

**Comment by rern** [comment, marked as answer]
With today's update, any slashes for CIFS share will be removed.
